### DISCUSSION TITLE
No detection on small images

### DISCUSSION THREAD

**Discussion by underthesand** [mindee/doctr#1057] [Q&A]
On small images there is no detection but if I zoom out there is one.

how i can solve this problem please ?

Édit : As DocTR hasnt been trained on small chucks of texts the solutions are custom training or only using the recognition model if we already know text position. 
Source https://github.com/mindee/doctr/issues/639

Big
![Screenshot from 2022-09-10 10-22-00](https://user-images.githubusercontent.com/71890227/189475288-f808638d-4dc7-4db4-b000-2d276c58da1b.png)

Small
![Screenshot from 2022-09-10 10-20-50](https://user-images.githubusercontent.com/71890227/189475451-80fc16ea-884c-4d47-a588-c169b43dabcb.png)

**Comment by felixdittrich92** [comment]
Hi @underthesand :wave:, another option would be to pad small images if you want to use the whole pipeline.

**Reply by frgfm** [reply]
FYI @underthesand this is a workaround, not the longterm solution we'll work on :+1:
